### PR TITLE
Update convert-user-mailbox-to-shared-mailbox.md

### PR DESCRIPTION
**Pull request by msbemba**
option to convert the user mailbox to a shared mailbox is no longer available in the m365 admin center.

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9  Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/2833

File: microsoft-365/admin/email/convert-user-mailbox-to-shared-mailbox.md [hunk 1]
```diff
@@ -46,46 +46,9 @@ When you convert a user's mailbox to a shared mailbox, all of the existing email
...
-1. In the admin center, go to the **Users** \> <a href="https://go.microsoft.com/fwlink/p/?linkid=850628" target="_blank">Active users</a> page.
-
-2. Select the user whose mailbox you want to convert.
-
-3. In the right pane, expand **Mail Settings**. Next to **More settings**, select **Convert to shared mailbox**.
-
-::: moniker-end
-
-
 If the mailbox is smaller than 50 GB, you can [remove the license from the user](../manage/remove-licenses-from-users.md), and stop paying for it. Don't delete the user's old mailbox. The shared mailbox needs it there as an anchor. If you are converting the mailbox of an employee that is leaving your organization, you should take additional steps to make sure that they cannot log in anymore. See [Remove a former employee from Microsoft 365](../add-users/remove-former-employee.md).
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/email/convert-user-mailbox-to-shared-mailbox.md]:
```suggestion
```

Comment (JohanFreelancer9) [reply microsoft-365/admin/email/convert-user-mailbox-to-shared-mailbox.md]: This is duplication of Number 4 above, and I would suggest removing it.

File: microsoft-365/admin/email/convert-user-mailbox-to-shared-mailbox.md [hunk 2]
```diff
@@ -46,47 +46,9 @@ When you convert a user's mailbox to a shared mailbox, all of the existing email
...
-
-1. In the admin center, go to the **Users** \> <a href="https://go.microsoft.com/fwlink/p/?linkid=834822" target="_blank">Active users</a> page.
-
-2. Select the name of the user whose mailbox you want to convert.
-
-3. Reset the user's password.
-
-   > [!NOTE]
+> [!NOTE]
    > It's not required to reset the user's password during mailbox conversion. However, if the password is not reset, **the original username and password continue to work** after the mailbox conversion is finished.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/email/convert-user-mailbox-to-shared-mailbox.md]:
```suggestion
> It's not required to reset the user's password during mailbox conversion. However, if the password is not reset, **the original username and password continue to work** after the mailbox conversion is finished.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

File: microsoft-365/admin/email/convert-user-mailbox-to-shared-mailbox.md [hunk 3]
```diff
@@ -46,47 +46,9 @@ When you convert a user's mailbox to a shared mailbox, all of the existing email
...
-2. Select the user whose mailbox you want to convert.
-
-3. In the right pane, expand **Mail Settings**. Next to **More settings**, select **Convert to shared mailbox**.
-
-::: moniker-end
-
+> [!NOTE]
+> It's not required to reset the user's password during mailbox conversion. However, if the password is not reset, **the original username and password continue to work** after the mailbox conversion is finished.
 
-If the mailbox is smaller than 50 GB, you can [remove the license from the user](../manage/remove-licenses-from-users.md), and stop paying for it. Don't delete the user's old mailbox. The shared mailbox needs it there as an anchor. If you are converting the mailbox of an employee that is leaving your organization, you should take additional steps to make sure that they cannot log in anymore. See [Remove a former employee from Microsoft 365](../add-users/remove-former-employee.md).
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/email/convert-user-mailbox-to-shared-mailbox.md]:
```suggestion
```

Comment (JohanFreelancer9) [reply microsoft-365/admin/email/convert-user-mailbox-to-shared-mailbox.md]: Please just remove the extra blank line.

**Comment by yogkumgit** [comment]
@cmcatee-MSFT Please merge this PR. Thanks

**Pull request merged**